crates/bevy_ecs/src/schedule/ambiguity.rs

```diff
@@ -0,0 +1,46 @@
+use core::{
+    fmt::Write as _,
+    ops::{Deref, DerefMut},
+};
+
+use alloc::string::{String, ToString};
+use bevy_platform::collections::HashSet;
+
+use crate::{
+    component::{ComponentId, Components},
+    resource::Resource,
+};
+
+/// List of [`ComponentId`]s to ignore when reporting system order ambiguity conflicts.
+#[derive(Resource, Default)]
+pub struct IgnoredAmbiguities(pub HashSet<ComponentId>);
```

Comment:
Each schedule is going to produce it's own ambiguities. Shouldn't this be like `HashMap<ScheduleLabel, HashSet<ComponentId>>`? or maybe a component might be better. Since you can store schedules outside of the `Schedules` resource.